test/vfd_swmr_group_writer.c

```diff
@@ -34,28 +34,32 @@ typedef struct {
 	unsigned int nsteps;
         unsigned int update_interval;
         bool use_vfd_swmr;
+        bool use_named_pipes;
 } state_t;
 
 #define ALL_HID_INITIALIZER                                                                                  \
     (state_t)                                                                                                \
     {                                                                                                        \
         .file = H5I_INVALID_HID, .one_by_one_sid = H5I_INVALID_HID, .filename = "",                          \
         .filetype = H5T_NATIVE_UINT32, .asteps = 10, .csteps = 10, .nsteps = 100, .update_interval = READER_WAIT_TICKS,   \
-        .use_vfd_swmr = true                                                                                 \
+        .use_vfd_swmr = true,                                                                                \
+        .use_named_pipes = true                                                                              \
     }
 
 static void
 usage(const char *progname)
 {
 	fprintf(stderr, "usage: %s [-S] [-a steps] [-b] [-c]\n"
-                "    [-n iterations] [-u numb_ticks]\n"
+                "    [-n iterations] [-p] [-q] [-u numb_ticks]\n"
 		"\n"
 		"-S:	               do not use VFD SWMR\n"
 		"-a steps:	       `steps` between adding attributes\n"
 		"-b:	               write data in big-endian byte order\n"
 		"-c steps:	       `steps` between communication between the writer and reader\n"
                 "-n ngroups:           the number of groups\n"
+		"-p:	               do not use named pipes, mainly for running the writer and reader seperately\n"
```

Comment:
I use "-N" in my new test for "do not use named pipes".
Maybe we can come up with a name for this option that is suitable for all the tests.

Reply:
I changed it to `-N`, too.